clients/python/girder_client/__init__.py

```diff
@@ -1401,7 +1401,7 @@ def loadLocalMetadata(self, dest):
         try:
             with open(os.path.join(dest, '.girder_metadata'), 'r') as fh:
                 self.localMetadata = json.loads(fh.read())
-        except (IOError, OSError):
+        except (OSError if six.PY3 else (IOError, OSError)):
```

Comment:
Does this change anything in practice?

Reply:
Nope, the behavior at runtime should be the same.

Reply:
I made the change in this way (as opposed to `noqa`) with an eye towards eventually dropping Python 2 support. When the conditional is elided under Python 3, it's now catching the correct exception, as opposed to the redundancy remaining there indefinitely and being ignored.

Reply:
Fair enough.